src/main/openapi/ds-present-openapi_v1.yaml

```diff
@@ -172,6 +170,60 @@ paths:
                   SolrJSONDocuments representation of content, directly indexable in Solr.
                   See [Solr JSON Formatted Index Updates](https://solr.apache.org/guide/8_8/uploading-data-with-index-handlers.html#json-formatted-index-updates)
                 type: string
+
+  /recordsraw:
+    get:
+      tags:
+        - '${project.name}'
+      summary: 'Extract raw records from a collection after a given mTime.'
+      description: |
+        The extracted records are returned in sorted order by mTime increasing. By changing the parameter `maxRecords`, 
+        the size of the result can be defined. Records marked with a delete flag will also be returned. Records are returned 
+        in a raw storage record.
```

Comment:
Or more precisely: The raw records are returned wrapped in JSON. The caller does not necessarily know what raw storage format is.